### ISSUE TITLE
Polymorphic relationship (get polymorph parent from child)

### ISSUE THREAD

**Issue by mwei0210** [go-pg/pg#841]
Is there a way to retrieve the polymorph parent? take the [documentation](https://github.com/go-pg/pg/wiki/Model-Definition) for example: 

Is it possible to retrieve "Book" or "Translation" from "Comment"?

Thanks!

**Comment by vmihailenco** [comment]
No, it is not possible. I can't say if it is going to be added any time soon because it is too different from other relations and requires some work.

**Comment by mwei0210** [comment]
Got it thanks.
